Demos/Demos.pm

```diff
@@ -189,19 +192,20 @@ Each demo module must provide these class methods:
 
 =item info
 
-Return a two-element list: a single keyword (probably lower-case),
-and a short description of the demo.
+Return a two-element list: of strings: a single keyword (probably
```

Comment:
A tiny nit: you say "list:", and the ":" is unnecessary. One could argue that the description should make it very clear they are strings, but I don't mind being more explicit.